health-services/referralmanagement/pom.xml

```diff
@@ -46,12 +46,12 @@
 		<dependency>
 			<groupId>org.egov.common</groupId>
 			<artifactId>health-services-common</artifactId>
-			<version>1.0.18-SNAPSHOT</version>
+			<version>1.0.20-dev-SNAPSHOT</version>
```

Comment:
remove dev, update library and merge to master .

Reply:
done